_api-reference/profile.md

```diff
@@ -265,8 +284,13 @@ Field | Description
 :--- | :--- 
 `name` | The collector name. In the [example response](#example-response), the `collector` is a single `SimpleTopScoreDocCollector`---the default scoring and sorting collector.
 `reason` | Contains a description of the collector. For possible field values, see [Collector reasons](#collector-reasons).
-`time_in_nanos` | A wall-clock time, including timing for all children.
+`time_in_nanos` | The total elapsed time for this collector, in nanoseconds. For concurrent segment search, `time_in_nanos` is the total amount of time across all slices (the difference between the last completed slice execution end time and the first slice execution start time).
 `children` | If a collector has subcollectors (children), this field contains information about the subcollectors.
+`max_slice_time_in_nanos`	|The maximum amount of time taken by any slice, in nanoseconds.	
```

Comment:
Should we mention these stats fields only present in the concurrent segment search response? These stats won't show in a non-concurrent segment search response.